tests/lib/rules/jsx-indent.js

```diff
@@ -137,18 +157,23 @@ ruleTester.run('jsx-indent', rule, {
     ].join('\n'),
     options: [2],
     parserOptions: parserOptions,
-    errors: [{message: 'Expected indentation of 2 space characters but found 9.'}]
+    errors: [
+      {message: 'Expected indentation of 2 space characters but found 9.'},
+      {message: 'Expected indentation of 2 space characters but found 9.'}
```

Comment:
You're right, it should be only one error. The `openingIndent` and `closingIndent` values are correct, but I think the `report()` is called with the wrong node. I tried to pass the `openingElement` and the `closeingElement` but still it's not working. Do you have anything in mind how to solve this?